engine/src/gldrv/gl_globals.h

```diff
@@ -39,6 +39,16 @@ const static bool GFX_BUFFER_MAP_UNMAP = false;
     #endif
 #endif
 
+#ifndef APIENTRY
+#define APIENTRY
+#endif
+#ifndef APIENTRYP
+#define APIENTRYP APIENTRY *
+#endif
+#ifndef GLAPI
+#define GLAPI extern
+#endif
```

Comment:
I'm curious where you got this block of code from.

Reply:
oh, that wasnt supposed to be in there. that was trying to get gl to work on my system. still curious why osx works on ci at all